src/package.xml

```diff
@@ -3254,7 +3254,6 @@
         <members>stgTools</members>
         <members>stgUnknownError</members>
         <members>stgValidationHHAccountHHRules</members>
-        <members>gsVideoHeaderLength</members>
```

Comment:
@dcaballeroSf , @force2b :  The `cci task run update_package_xml` removed a duplicate mention of the `gsVideoHeaderLength` custom label in the package.xml. The Beta Upload package failed due to other reasons, so not sure if this is an issue with the current package.xml version in the master.
CC @davidmreed